src/config/config_def.c

```diff
@@ -1090,16 +1098,18 @@ static const WT_CONFIG_CHECK confchk_wiredtiger_open_usercfg[] = {
 	    NULL, NULL,
 	    confchk_wiredtiger_open_eviction_subconfigs, 2 },
 	{ "eviction_checkpoint_target", "int",
-	    NULL, "min=0,max=99",
+	    NULL, "min=0,max=10TB",
 	    NULL, 0 },
 	{ "eviction_dirty_target", "int",
-	    NULL, "min=1,max=99",
+	    NULL, "min=1,max=10TB",
 	    NULL, 0 },
 	{ "eviction_dirty_trigger", "int",
-	    NULL, "min=1,max=99",
+	    NULL, "min=1,max=10TB",
+	    NULL, 0 },
+	{ "eviction_target", "int", NULL, "min=10,max=10TB", NULL, 0 },
```

Comment:
can make eviction_target also same as other parameters, in multiple lines.

Reply:
this file is auto generated .